lib/chat/toolChains/createNewArtistToolChain.ts

```diff
@@ -16,15 +16,28 @@ export const createNewArtistToolChain: ToolChainItem[] = [
   { toolName: "get_spotify_album" },
   {
     toolName: "create_knowledge_base",
-    system: "Create a knowledge base with the information you have gathered.",
+    system:
+      "Aggregate key, relevant findings from: artist_deep_research, spotify_deep_research, get_artist_socials, get_spotify_artist_top_tracks, get_spotify_artist_albums, get_spotify_album. De-duplicate, normalize names, and keep only high-signal facts. Do NOT include links, IDs, or citations.\n\nOrganize into:\n- Profile\n- Socials\n- Top Tracks\n- Albums\n- Notable Facts\n\nKeep concise (≈300–500 words total). Omit empty sections. Pass the compiled text as knowledgeBaseText.",
```

Comment:
how does changing the system prompt for the `create_knowledge_base` tool do anything?
- what `input props` will change?
- what `function calling` will change?
- what `return values` will change?
In my opinion this doesn't do anything and `system` can be removed from `create_knowledge_base` 🚯